src/plugins/data_source_management/public/plugin.ts

```diff
@@ -24,6 +25,7 @@ export interface DataSourceManagementSetupDependencies {
 
 export interface DataSourceManagementStartDependencies {
   data: DataPublicPluginStart;
+  dataSource: DataSourceStart;
 }
 
 export type DataSourceManagementSetup = DataSourceManagementServiceSetup;
```

Comment:
we should define the order of data source and credential management in the management page, e.g.:
* credentials
* data sources
* index patterns
* saved objects
* advance settings